### PR TITLE
request-history-tests

### PR DESCRIPTION
**Pull request by annachemic**
- added FileId methods
- fixed TEST-179 https://drive.google.com/file/d/1QgrAcL0TJy0HvgMoyzdl5XmUClIHSih_/view?usp=sharing
- fixed TEST-189 https://drive.google.com/file/d/1SGUbUwlv2WOKnwMjpwMySqzZb1QCSw1-/view?usp=sharing
- fixed @filterfileid  https://drive.google.com/file/d/1pDGorh6lv_i6JWL8njp_YqPCcGYTE1SW/view?usp=sharing (fails due to unavailability of Add Id button)

### CODE REVIEW

File: src/pages/request-history.page.js [hunk 1]
```diff
@@ -474,5 +467,23 @@ module.exports = {
...
+            col= 4;
+        }
+   //     if (res.isUUID()) return 2;
+        else col= 3;
+        return col;
+    },
+
+    isUUID (uuid) {
+    let s = "" + uuid;
+    s = s.match('^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$');
```
Comment (NadHodeGW) [inline src/pages/request-history.page.js:468]: can you explain this please s = s.match('^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}$'); what is the use here?

File: src/utils/helper.js [hunk 2]
```diff
@@ -55,6 +55,10 @@ class MyHelper extends Helper {
         console.log('Skipping operation as there was a problem getting the cell');
     }
 }
+
+        compareThatEqual(word1, word2){
```
Comment (NadHodeGW) [inline src/utils/helper.js:81]: what is this for? what is it needed for?  compareThatEqual

File: src/utils/helper.js [hunk 3]
```diff
@@ -36,15 +36,21 @@ class MyHelper extends Helper {
...
-        for (let i = 0; i < rowCount; i++) {
-            const text = await page.$eval(
-                `${tableRows}:nth-child(${i + 1}) th:nth-child(${col})`,
-                (e) => e.innerText
-            )
-            if (this.compareThatEqual(text, val)) {
-                console.log('The result list shows required files with the filter: ' + text);
+        try {
+            let rowCount = await page.$$eval(tableRows, rows => rows.length);
+            if (rowCount<1) {
```
Comment (NadHodeGW) [inline src/utils/helper.js]: what are you doing? why if (rowCount<1) if anything it should be >1; please do not change any functions unless agreed with me.

File: src/pages/request-history.page.js [hunk 4]
```diff
@@ -251,12 +251,12 @@ module.exports = {
...
             if (element) {
-                I.say("No Transaction Data Found")
+                await I.say("No Transaction Data Found")
             } else {
-                I.say("The table data is available")
+                await I.say("The table data is available")
             }
         } catch (e) {
-            I.say('errors')
+            await I.say('errors')
```
Comment (NadHodeGW) [inline src/pages/request-history.page.js]: why are you changing this to a async function, what is the use of await in front of I.say? none; please put back to previous state

**Review by NadHodeGW — CHANGES_REQUESTED** [review]
Please see comments in files

File: src/utils/helper.js (hunk 3 above)
Comment (annachemic) [reply src/utils/helper.js]: I think it's outdated, I don't see this row anymore

File: src/pages/request-history.page.js (hunk 4 above)
Comment (annachemic) [reply src/pages/request-history.page.js]: Outdated

File: src/utils/helper.js (hunk 2 above)
Comment (annachemic) [reply src/utils/helper.js:81]: It is a function that compare two strings ignoring the case
See, we have docx (small letters), but Safe (with the capital first letter), but in the field we see SAFE. 
I want to resolve this conflict and work with values universally, without thoughts about its case

File: src/pages/request-history.page.js (hunk 1 above)
Comment (annachemic) [reply src/pages/request-history.page.js]: It is deleted now, it's obsolete
It was a function to check UUIDs (fileIDs)

**Review by NadHodeGW — APPROVED** [review]

**Pull request merged**